modules/ROOT/pages/proxy-latest-concept.adoc

```diff
@@ -1,30 +1,66 @@
+Before you deploy your API proxy, ensure that you complete the following requirements based on where you are deploying the proxy:
 
-In most cases, the proxy you generate in API Manager is suitable for deployment. However, you can modify the proxy to log data to a file or send data to a Splunk account with the Anypoint Splunk Connector, for example.
+* Hybrid: 
+** For private servers, you must have access to a Mule runtime engine (mule) instance that has been configured for your organization. 
++
+For information on how to configure a Mule instance as a server, see xref:runtime-manager::servers-create.adoc[register a server in Runtime Manager].
+** You must register your private server with a Mule agent that is compatible with your Mule instance.
++
+For Mule versions 3.8 and later, use Mule Agent 1.9.0. For Mule 4.x, use the Mule Agent specified in the release notes. 
+* Runtime Fabric
+** You must have the Manage APIs Configuration and Deploy API Proxies permissions on your account.
+** You must be using Mule versions 3.8.7, 3.9.1, or 4.1.2, or later.
+* CloudHub:
++
+Currently, no prerequisites are required for CloudHub
 
-You can import a proxy that you download from API Manager into Studio as a Mule Deployable Archive. You can then modify the proxy in Studio.
+//Kim, IMO, not mentioning that no prerequisites are required for CH might be confusing for the users, so I addedd it. What do you think?
 
-If you configured an endpoint with a proxy based on Mule 3 to use the HTTPS scheme, you need to modify a proxy to add HTTPS credentials following the xref:mule-runtime::build-an-https-service.adoc[Build an HTTPS Service] instructions. +
-Additionally, if you configured a proxy based on Mule 4, you can build an HTTPS proxy using a TLS context stored in Anypoint Security's secrets manager following the xref:building-https-proxy.adoc[Building an HTTPS API Proxy] instructions.
+== Deploy your API Proxy
 
+To deploy your proxy:
 
+. From Anypoint Platform, select *API Manager* > *Manage API* > *Manage API From Exchange*.
+. In *API Name*, search for the name of the asset as saved in Exchange.
++
+The *Asset type*, *API version*, and *Asset version* fields for the API are auto-populated from Exchange.
+. From *Managing type*, select *Endpoint with proxy* and select the *Proxy deployment target* for the proxy, *CloudHub*, *Hybrid*, or *Runtime Fabric*:
++
+image:select-deployment-target.png[Select Managing type and Deployment Target]
 
-Now, you're ready to deploy an auto-generated proxy.
+<1> Search for APIs from Exchange.
+<2> Select a proxy deployment target.
+. If your Mule target is Mule 4, check *Mule version* or leave it unchecked if you are using Mule 3.
++
+If you are deploying your API to a Mule 4 environment and want to enable the proxy to support HTTPS requests, you must first xref:building-https-proxy.adoc[build HTTPS API proxies]. 
```

Comment:
```suggestion
If you are deploying your API to a Mule 4 environment and want to enable the proxy to support HTTPS requests, first xref:building-https-proxy.adoc[build HTTPS API proxies]. 
```

Reply:
Again, this is interesting, but is it crucial to THIS task of deploying?

Reply:
Yes it is needed!

Reply:
Done! Thanks!